nixos/modules/services/networking/nordvpn.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  config,
+  pkgs,
+  proc-ps,
+  ...
+}:
+
+let
+  cfg = config.services.nordvpn;
+  nordvpnPkg = cfg.package;
+  nordvpn = "nordvpn";
+  myPkgs =
+    with pkgs;
+    [
+      iptables
+      iproute2
+      procps
+      sysctl
+      e2fsprogs
+    ]
+    ++ [
+      cfg.package
+    ];
+in
+
+{
+  options.services.nordvpn = {
+    enable = lib.mkEnableOption "Enable NordVPN";
+    package = lib.mkPackageOption pkgs "nordvpn" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+
+    users.groups."${nordvpn}" = { };
+
+    users.users."${nordvpn}" = {
+      isNormalUser = true;
+      group = nordvpn;
+      home = "/home/${nordvpn}";
```

Comment:
Why is this not a system user?
And is the nordvpn variable used when it's just "nordvpn"?
The home directory probably also shouldn't be at /home